internal/store/leaf/reencrypt.go

```diff
@@ -41,7 +41,7 @@ func (s *Store) reencrypt(ctx context.Context) error {
 		jobs := make(chan string)
 		// We use a logger to write without race condition on stdout
 		logger := log.New(os.Stdout, "", 0)
-		out.Printf(ctx, "Starting reencrypt")
+		out.Println(ctx, "Starting reencrypt")
```

Comment:
There is no out.Println. Use out.Printf, please.

Reply:
Used `out.Print()` instead, it also exists